### PR TITLE
README: document a Deployment section instead of Running

### PR DESCRIPTION
**Pull request by haosanzi**
Fixes: #241

Signed-off-by: Shirong Hao <shirong@linux.alibaba.com>

### CODE REVIEW

**Comment by jiazhang0** [comment]
Still need changes to clarify what the differences are between containerd and dockerd integration. @haosanzi

File: README.md [hunk 1]
```diff
@@ -100,6 +100,10 @@ sudo apt-get install rune shim-rune sgx-tools
 
 Inclavare Containers can be integrated with dockerd and containerd.
 
+- Inclavare Containers integrated with dockerd can be deployed on docker scenarios, you need to install enclave image into container image so that various enclave runtimes can be launched by `rune` through well-defined [Enclave Runtime PAL API](rune/libenclave/internal/runtime/pal/spec.md).
+
+- Inclavare Containers integrated with containerd can be deployed on Confidential Computing Kubernetes Cluster scenarios. Because `shim-rune` and `rune` can compose a basic enclave containerization stack, container image don't need to contain enclave runtime.
```
Comment (hustliyilin) [inline README.md:105]: Attentation your spelling please, you should use `container image doesn't` or `container images don't`, You should check grammars before committing your code.

Comment (hustliyilin) [reply README.md:105]: You can use [grammarly](https://www.grammarly.com/) to check your spelling before committing your docs.

File: README.md [hunk 2]
```diff
@@ -100,6 +100,10 @@ sudo apt-get install rune shim-rune sgx-tools
 
 Inclavare Containers can be integrated with dockerd and containerd.
 
+- Inclavare Containers integrated with dockerd can be deployed on docker scenarios, you need to install enclave image into container image so that various enclave runtimes can be launched by `rune` through well-defined [Enclave Runtime PAL API](rune/libenclave/internal/runtime/pal/spec.md).
```
Comment (jiazhang0) [inline README.md:103]: Change to:

```
The former targets using docker to deploy Inclavare Containers. Specifically, you need to install the preferred enclave runtime when building container image, and then launch the enclave runtime through `rune` and [enclave runtime specific PAL](docs/design/terminology.md#enclave-runtime-pal).
```

File: README.md [hunk 3]
```diff
@@ -100,6 +100,10 @@ sudo apt-get install rune shim-rune sgx-tools
...
+
+- Inclavare Containers integrated with containerd can be deployed on Confidential Computing Kubernetes Cluster scenarios. Because `shim-rune` and `rune` can compose a basic enclave containerization stack, container images don't need to contain enclave runtime.
```
Comment (jiazhang0) [inline README.md:105]: Chang to:

```
The latter targets using K8s to deploy Inclavare Containers. In this scenario, `shim-rune` and `rune` can compose an enclave containerization stack, so enclave runtime is not required and installed when building container image,  providing with the same experience as ordinary containers.
```

File: README.md [hunk 4]
```diff
@@ -136,16 +140,20 @@ Add the assocated configurations for shim-rune in containerd config file, e.g, `
...
-## Confidential Computing Kubernetes Cluster
+## containerd
 
 Please refer to [this guide](docs/develop_and_deploy_hello_world_application_in_kubernetes_cluster.md) to deploy an enclave container in a Kubernetes cluster.
 
-## Occlum LibOS
+[The reference container images](https://hub.docker.com/u/inclavarecontainers) are available for the demonstration purpose to show how a Confidential Computing Kubernetes Cluster with Inclavare Containers works. Currently, web application demos based on OpenJDK 11, [Dragonwell](http://dragonwell-jdk.io/), and Golang are provided. 
 
-Please refer to [this guide](docs/Running_Occlum_with_Docker_and_OCI_Runtime_rune.md) to run [Occlum](https://github.com/occlum/occlum) with `rune`.
+**Note that we don't provide any reference images containing enclave runtime.**
```
Comment (jiazhang0) [inline README.md:159]: Change to:

```
All reference container images don't contain enclave runtime. They are only used for the deployment with containerd.
```

**Review by jiazhang0 — CHANGES_REQUESTED** [review]

**Review by hustliyilin — COMMENTED** [review]
It’s best not to use ``` in the commit title and body.

**Review by hustliyilin — APPROVED** [review]

**Pull request merged**